azafea/event_processors/endless/metrics/events/_base.py

```diff
@@ -211,7 +211,7 @@ class UnknownAggregateEvent(AggregateEvent, UnknownEvent):
 class SequenceEvent(MetricEvent):
     __abstract__ = True
 
-    started_at = Column(DateTime(timezone=True), nullable=False)
+    started_at = Column(DateTime(timezone=True), nullable=False, index=True)
```

Comment:
While we're here can we add the same on `occured_at` for other event types?

Reply:
Done!